projects/plugins/jetpack/_inc/client/main.jsx

```diff
@@ -960,14 +960,69 @@ export default connect(
 	} )( withRouter( Main ) )
 );
 
+// eslint-disable-next-line jsdoc/require-returns-check
+/**
+ * Determines the page order of My Jetpack, Activity Log, Dashboard, and Settings in the left sidebar.
+ * @return {object} Object with keys for each page and values for the order of the page in the sidebar.
+ */
+function jetpackPageOrder() {
+	const jetpackParentMenu = document.querySelector( '#toplevel_page_jetpack' );
+	const pageOrder = {};
+
+	if ( jetpackParentMenu ) {
+		const jetpackSubMenu = jetpackParentMenu.querySelector( '.wp-submenu' );
+
+		if ( jetpackSubMenu ) {
+			const subMenuItems = jetpackSubMenu.querySelectorAll( 'li:not(.wp-submenu-head) a' );
+
+			const urlPatterns = [
+				{
+					key: 'dashboard',
+					pattern: '/wp-admin/admin.php?page=jetpack#/dashboard',
+					matchType: 'end',
+				},
+				{
+					key: 'activityLog',
+					pattern: 'https://jetpack.com/redirect/?source=cloud-activity-log-wp-menu',
```

Comment:
It seems to me this isn't needed, as I don't see how we'd be able to see the Activity Log menu item as a selected item, since it is an external link. Leaving for now to ensure I'm not introducing any bugs but this can also be removed if there is no known reason to keep it.